update-requirements

```diff
@@ -23,6 +23,7 @@ EXCLUDE_NPM_PACKAGES = [
     'babel-plugin-dynamic-import-node',
     'coveralls',
     'cross-env',
+    'eslint-plugin-spellcheck',
```

Comment:
This isn't really needed since `eslint` is in `EXCLUDE_NPM_PREFIXES`. That was easier than keeping track of all the plugins.

Reply:
cc @evgeni

Reply:
oh, then I shut up what I said in the foreman PR :D